pyQuARC/code/custom_validator.py

```diff
@@ -9,20 +11,20 @@ def __init__(self):
         super().__init__()
 
     @staticmethod
-    @if_arg
     def ends_at_present_flag_logic_check(
         ends_at_present_flag, ending_date_time, collection_state
     ):
-        value = ends_at_present_flag.lower()
         collection_state = collection_state.upper()
-
-        valid = (
-            value == "true"
-            and not (ending_date_time) or collection_state == "ACTIVE"
-        ) or (
-            value == "false"
-            and ending_date_time or collection_state == "COMPLETE"
-        )
+        if ends_at_present_flag == None:
+            valid = True
+        else:
```

Comment:
`if not(valid := ends_at_present_flag == None):`

Reply:
smort